build.sbt

```diff
@@ -39,6 +39,18 @@ lazy val monixLogbackHttp4s: Project = (project in file("monix-logback-http4s"))
       "io.monix" %% "monix" % "3.1.0",
       "ch.qos.logback" % "logback-classic" % "1.2.3",
       scalaTest) ++ http4sDependencies(scalaVersion.value)
-    )
-
+  )
 
+lazy val zioLogbackHttp4s: Project = (project in file("zio-logback-http4s"))
+  .settings(commonSettings: _*)
+  .settings(
+    name := "zio-logback-http4s",
+    libraryDependencies ++= Seq(
+      "dev.zio" %% "zio" % "1.0.0-RC16",
+      "dev.zio" %% "zio-interop-cats" % "2.0.0.0-RC7",
+      "ch.qos.logback" % "logback-classic" % "1.2.3",
+      "org.http4s" %% "http4s-core" % "0.21.0-M5",
+      "org.http4s" %% "http4s-dsl" % "0.21.0-M5" % "test",
```

Comment:
Because `zio-interop-cats` doesn't support Cats 1.0, we cannot provide this project for Http4s `0.20`

Reply:
sure thing, I suppose we should update everywhere to 0.21. Bootzoooka is already using it (for similar reasons :) )

Reply:
Ok I'll do that.